examples/docker-without-nfs/assets/conda_store_config.py

```diff
@@ -57,11 +58,12 @@
 #         auth settings
 # ==================================
 c.CondaStoreServer.authentication_class = JupyterHubOAuthAuthentication
-c.JupyterHubOAuthAuthentication.jupyterhub_url = "https://conda-store.localhost"
+c.JupyterHubOAuthAuthentication.jupyterhub_url = "http://conda-store.localhost"
```

Comment:
I don't have the context on why this change is needed. Could you provide some insight about this?